lunaria.config.json

```
@@ -0,0 +1,57 @@
+{
+	"$schema": "./node_modules/@lunariajs/core/config.schema.json",
+	"repository": {
+		"name": "solidjs/solid-docs-next",
+		"branch": "main",
+		"hosting": "github"
+	},
+	"defaultLocale": {
+		"label": "English",
+		"lang": "en"
+	},
+	"files": [
+		{
+			"type": "universal",
+			"location": "src/routes/**/*.{mdx,md,tsx}",
```

Heads up this glob would also match the `src/routes/[...404]/.tsx` that doesn't look like should be translated? It uses a dictionary key, so we can probably just not add TS files into tracking here.
```suggestion
			"location": "src/routes/**/*.{mdx,md}",
```